packages/suite-web/package.json

```diff
@@ -9,8 +9,6 @@
         "copy-files": "yarn workspace @trezor/suite-data copy-static-files",
         "dev": "rimraf ./build && yarn workspace @trezor/suite-build run dev:web",
         "analyze": "ANALYZE=true yarn build",
-        "cert": "sh ./scripts/gencert.sh",
```

Comment:
it was supposed to help with debugging on android, but it wasn't updated after we get rid off Nextjs and it wasn't working since then and nobody noticed, so we decided to remove it – anybody could take it from git history once needed.